presto-native-execution/presto_cpp/main/PrestoServer.cpp

```diff
@@ -863,13 +863,25 @@ std::vector<std::string> PrestoServer::registerConnectors(
 void PrestoServer::unregisterConnectors() {
   PRESTO_SHUTDOWN_LOG(INFO) << "Unregistering connectors";
   auto connectors = facebook::velox::connector::getAllConnectors();
-  std::unordered_set<std::string> connectorIds;
```

Comment:
This was never populated with connector names. Hence the log at Line 869 was not printing any names.